add: 16165 code

hsilnam opened the pull request:
제가 걸그룹 마스텁니다!!

hsilnam committed:
add: 16165 code

hsilnam committed:
fix: 16493 code

hsilnam committed:
fix: 16493 code

hsilnam commented:
!! 최대 페이지수 드디어 풀다 !!
해당 일의 최대 페이지수를 구하고 업데이트해준다
Max(기존+현재(현재 포함O), 기존(현재 포함X))을 비교하면서 진행한다
- 앞에서부터 순차적으로 검사할 경우: 2차원 배열 (현재값 업데이트로 인해 전 값을 가지고 있어야하기 떄문)
- 뒤에서부터 순차적으로 검사할 경우: 1차원 배열 (업데이트 하는 값 전 값을 비교하는데 영향을 안 미치기 때문)
=> 1차원 배열로 풀었습니다!!

hsilnam committed:
add
add: 17245 code

mikaniz reviewed (approved):
수고하셨습니다~~~~

sooyeon-kr reviewed (approved):
🫵🏻🫵🏻🫵🏻🫵🏻🫵🏻👍🏻👍🏻👍🏻👍🏻👍🏻

hsilnam committed:
add : 11279 code

hsilnam committed:
add: 23630

각 숫자(32비트)의 비트 별 1값을 cnt를 해서 가장 긴 큰값이 답
- &연산으로 1이 나오려면 전부 1로 이루어져야하고
- 비트는 0, 1로만 구성되어 있으니
- bit값이 1인 것에 대한 cnt를 세면 된다

Hyeon-Uk reviewed (approved):
걸그룹 마스터 현실!걸그룹 마스터 현실!걸그룹 마스터 현실!걸그룹 마스터 현실!
